assets/js/reducers/rootReducer.js

```diff
@@ -1,4 +1,4 @@
-const rootReducer = (state = [], action) => {
+const rootReducer = (state = {}, action) => {
```

Comment:
we could also just swap the state to being just the array of events if we think we won't need to store anything else there

Reply:
Is this default value for if there is no initial state?